src/Api/Dashboard/DashboardApi.tsx

```diff
@@ -0,0 +1,31 @@
+const serverUrl = "http://18.60.129.53/";
+
+export const primaryDashboard = async () => {
+  const options = {
+    method: "GET",
+  };
+  const url = `${serverUrl}primarydashboard`;
+  const response = await fetch(url, options);
+  const responseData = await response.json();
+  return { status: response.status, data: responseData.data };
+};
```

Comment:
While this is good, you aren't doing any error handling. Let's think about that

Reply:
Error handling is done inside each and every component based on the status of the response.